roles/ovirt-cluster-upgrade/tasks/main.yml

```diff
@@ -0,0 +1,128 @@
+---
+- block:
+  - name: Login to oVirt
+    ovirt_auth:
+      url: "{{ engine_url }}"
+      username: "{{ engine_user }}"
+      password: "{{ engine_password }}"
+      ca_file: "{{ engine_cafile | default(omit) }}"
+      insecure: "{{ engine_insecure | default(true) }}"
+    when: ovirt_auth is undefined
+    register: loggedin
+    tags:
+      - always
+
+  - name: Get hosts
+    ovirt_hosts_facts:
+      auth: "{{ ovirt_auth }}"
+      pattern: "cluster={{ cluster_name | mandatory }} and update_available=true"
```

Comment:
I think we should limit hosts:

1. By status
    - we should provide role parameter to override host statuses in which hosts should be upgraded
    - by default I think we should try to upgrade only hosts in Up, because other status usually require administrators attention
2. By host name
    - by default this parameter will be empty (meaning that all hosts can be upgraded), or if non-empty then only hosts mentioned in this list will be upgraded